domain-ee/ee-ep-merge-app/pyproject.toml

```diff
@@ -0,0 +1,11 @@
+# pyproject.toml
+[tool.pytest.ini_options]
+minversion = "6.0"
+addopts = "-rfesp"
+testpaths = [
+    "tests"
```

Comment:
Should the `"integration"` folder be added to `testpaths` in #2106? I'm assuming the answer is "not yet" since 5 of the integration tests aren't passing for me - is this blocked pending further API changes, and are we tracking that blocker?

Reply:
Great question! The integration tests folder is meant to be run via the [CI github action](https://github.com/department-of-veterans-affairs/abd-vro/actions/workflows/continuous-integration.yml)! Here's the latest [run](https://github.com/department-of-veterans-affairs/abd-vro/actions/runs/6552224404) for that branch.

As far as tracking the blocker of the hoppy-lib in PR #2044, I added a line to the PR description!